contracts/rootchain/RootChain.sol

```diff
@@ -149,29 +150,32 @@ contract RootChain is Ownable {
     // @param confirmSignatures confirm signatures sent by the owners of the inputs acknowledging the spend.
     // @notice `confirmSignatures` and `ConfirmSig0`/`ConfirmSig1` are unrelated to each other.
     // @notice `confirmSignatures` is either 65 or 130 bytes in length dependent on if input2 is used.
-    function startTransactionExit(uint256[3] txPos, bytes txBytes, bytes proof, bytes sigs, bytes confirmSignatures)
+    function startTransactionExit(uint256[3] txPos, bytes txBytes, bytes proof, bytes confirmSignatures)
         public
         payable
     {
-        RLPReader.RLPItem[] memory txList = txBytes.toRlpItem().toList();
-        require(txList.length == 17, "incorrect tx length");
+        RLPReader.RLPItem[] memory txList;
+        bytes[] memory sigList;
+        bytes memory encodedTxList;
+        (txList, sigList, encodedTxList) = extractTxListAndSigs(txBytes);
```

Comment:
Is a stack to deep error preventing decoding the txBytes directly within this function?

Reply:
the decoding steps are replicated throughout 3 functions, so I put them all into a separate function